### PR TITLE
fix: overdue milestone warning text fixes #765

### LINKED ISSUE #765: Overdue milestone alert is styled the wrong color on hover/selection
By default this alert is styled red and on hover/selection is white so it's easier to read. Our style is always red.

* **Browser**: chrome
* **Operating System**: win10
* **Screenshot**:

image

image

* **HTML of the section where the issue occurs**:

````html
<div data-filterable-for="context-milestone-filter-field" data-filterable-type="substring" role="menu">
  <div class="select-menu-item js-navigation-item last-visible navigation-focus" role="menuitem" aria-selected="true">
    <input type="radio" name="milestone" id="milestone_1234" value="1234">
    <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
    <div class="select-menu-item-text">
      <span class="select-menu-item-heading js-select-button-text">oct-28</span>
      <span class="description"><strong class="description-warning" title="October 28, 2018"><span class="mr-1"><svg class="octicon octicon-alert" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"></path></svg></span> Past due by 4 days</strong></span>
    </div>
  </div>
</div>
````

### PR DESCRIPTION
**Pull request by the-j0k3r**
also closes duplicate #813 (already manually closed as dupe)

Suggestions welcome

### CODE REVIEW

File: github-dark.css [hunk 1]
```diff
@@ -2898,10 +2898,10 @@
   .select-menu-item.navigation-focus > .octicon,
   .select-menu-item[aria-selected="true"] > .octicon, .text-white,
   .octicon.text-white,
-  .select-menu-item.selected.navigation-focus .description-warning,
   .suggester li.navigation-focus, .suggester li:hover, .suggester li:hover *,
   .suggester li[aria-selected="true"], .suggester li[aria-selected="true"] *,
-  details-menu .select-menu-item[aria-checked=true]:hover .description {
+  details-menu .select-menu-item[aria-checked=true]:hover .description,
+  .select-menu-item.navigation-focus .select-menu-item-text .description strong {
```
Comment (xt0rted) [inline github-dark.css]: Instead of adding `strong` to the selector it's probably better to move this rule to after the red one.

Comment (the-j0k3r) [reply github-dark.css]: You're right this requires more thought, the milestone being selected/non-selected matters as both need to be accounted for to resolve full issue.
Also position.

Comment (the-j0k3r) [reply github-dark.css]: Ive had a look, I cant get this to work unless the selector is changed, as for position this is well far past the red one as is.

Ive updated the selector a bit and still works, failing this solution, ive no idea.

**Comment by the-j0k3r** [comment]
Well push this in for time being at least its fixed.

**Pull request merged**